kamelets/telegram-sink.kamelet.yaml

```diff
@@ -60,8 +60,14 @@ spec:
         - urn:camel:group:credentials
       chatId:
         title: Chat ID
-        description: The Chat ID to where you want to send messages by default. Optional if the `chat-id` / `ce-chatid` header is used.
+        description: |-
+          The Chat ID to where you want to send messages by default. 
+          
+          The Chat ID must be provided for every message; either as a Configuration Option or a `chat-id` / `ce-chatid` header.
+          
+          If the Chat ID is not explicitly provided the default value is used which may lead to delivery failures.
         type: string
+        default: dummy 
```

Comment:
If you set a default value, it will be set on the endpoint by default.

Reply:
This is @valdar's preference. 

However I see your point too. Damn. Why do I always see both sides to things!?!?

The User is forced to either provide a _real_ value for the endpoint configuration or set its value in the header where it then takes precedence over the endpoint configuration. The default value is explicitly mentioned in the updated documentation and advises it _could_ lead to delivery failure.

It's a difficult situation: The parameter is _required_, it is not optional; but _where_ it can be set (endpoint or per message) means Camel fails to deploy the Kamelet if the User does not set a value in the Configuration Options when marked as `required`. 

We want to somehow impose configuration of a `chatId` on Users.

I'll chat to @valdar again.. the imposition is something we want in RHOC but seems contrary to both the underlying Camel Telegram Component and your opinion.

Reply:
OK. I've restored the `required` nature of the `chatId` parameter and hopefully made the documentation bullet proof!